mpi-proxy-split/test/large_async_p2p.c

```diff
@@ -0,0 +1,48 @@
+#include <mpi.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <unistd.h>
+
+#define MSG_SIZE 64*1024 // large message
+#define MSG_SIZE 64 // small message
```

Comment:
```
#define MSG_SIZE 64*1024 // large message
#ifndef MSG_SIZE
# define MSG_SIZE 64 // small message
#endif
```

1. Some compilers will complain about re-defining MSG_SIZE without this.
The default version uses a large message in this version.  But I think this is a good thing, since that's what caused the original bug.
2. Should we raise the large message size to `256*1024`, to make it more future-proof?  In the future, maybe the optimized hardware considers even `64*1024` to be a small message that can use an EAGER policy.

Reply:
This is a mistake. I was thinking of commenting out one of two definitions.